recipes/slearn/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "slearn" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+
+source:
+  fn: slearn-0.2.4.tar.gz
+  url: https://files.pythonhosted.org/packages/f8/dd/3bd36708e78802bcab553e7b4bfc15a4a433e26fbe5f5911b1f2533fe8cd/slearn-0.2.4.tar.gz
+  sha256: e58e210667b3f96d6ec494778146708daaf3d4cdd318a320694c93db30b4743d
+
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - python >=3.6
+    - pip
+    - pandas
+    - numpy 
+    - setuptools
+    - scipy
+    - lightgbm
+    - scikit-learn
+
+
+  run:
+    - python >=3.6
+    - scikit-learn
+    - pandas
+    - numpy
+    - scipy
+    - lightgbm
+    
+    
+test:
+  imports:
+    - slearn
+
+
+about:
+  home: https://github.com/nla-group/slearn
+  summary: 'A package linking symbolic representation with scikit-learn machine learning for time series prediction.'
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  dev_url: https://github.com/nla-group/slearn
+
+extra:
+  recipe-maintainers:
+    - nla-group
```

Comment:
Please specify a github user and not an organization for the maintainers list.

Reply:
Thanks, done!